rules/linux/at_command.yml

```diff
@@ -0,0 +1,26 @@
+title: Scheduled Task/Job At
+id: d2d642d7-b393-43fe-bae4-e81ed5915c4b
+status: stable
+description: Detects the use of at/atd
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1053.001/T1053.001.md
+logsource:
+    product: linux
+detection:
+    selection:
+        CommandLine|contains:
+          - ' at '
+          - ' atd '
+          - 'which atd'
+          - 'which at'
+          - 'systemctl status atd'
+          - 'service atd status '
```

Comment:
these are test commands for preparation/checks for binaries installation on the system, for the further test execution. so it's not related to the actual threat emulation %) you can remove this part